subprojects/packagefiles/oatpp/test/meson.build

```diff
@@ -0,0 +1,122 @@
+    'oatpp/core/parser/CaretTest.cpp',
+    'oatpp/core/parser/CaretTest.hpp',
+    'oatpp/core/provider/PoolTemplateTest.cpp',
+    'oatpp/core/provider/PoolTemplateTest.hpp',
+    'oatpp/core/provider/PoolTest.cpp',
+    'oatpp/core/provider/PoolTest.hpp',
+    'oatpp/encoding/Base64Test.cpp',
+    'oatpp/encoding/Base64Test.hpp',
+    'oatpp/encoding/UnicodeTest.cpp',
+    'oatpp/encoding/UnicodeTest.hpp',
+    'oatpp/network/ConnectionPoolTest.cpp',
+    'oatpp/network/ConnectionPoolTest.hpp',
+    'oatpp/network/UrlTest.cpp',
+    'oatpp/network/UrlTest.hpp',
+    'oatpp/network/monitor/ConnectionMonitorTest.cpp',
+    'oatpp/network/monitor/ConnectionMonitorTest.hpp',
+    'oatpp/network/virtual_/InterfaceTest.cpp',
+    'oatpp/network/virtual_/InterfaceTest.hpp',
+    'oatpp/network/virtual_/PipeTest.cpp',
+    'oatpp/network/virtual_/PipeTest.hpp',
+    'oatpp/parser/json/mapping/DTOMapperPerfTest.cpp',
+    'oatpp/parser/json/mapping/DTOMapperPerfTest.hpp',
+    'oatpp/parser/json/mapping/DTOMapperTest.cpp',
+    'oatpp/parser/json/mapping/DTOMapperTest.hpp',
+    'oatpp/parser/json/mapping/DeserializerTest.cpp',
+    'oatpp/parser/json/mapping/DeserializerTest.hpp',
+    'oatpp/parser/json/mapping/EnumTest.cpp',
+    'oatpp/parser/json/mapping/EnumTest.hpp',
+    'oatpp/parser/json/mapping/UnorderedSetTest.cpp',
+    'oatpp/parser/json/mapping/UnorderedSetTest.hpp',
+    'oatpp/web/ClientRetryTest.cpp',
+    'oatpp/web/ClientRetryTest.hpp',
+    'oatpp/web/FullAsyncClientTest.cpp',
+    'oatpp/web/FullAsyncClientTest.hpp',
+    'oatpp/web/FullAsyncTest.cpp',
+    'oatpp/web/FullAsyncTest.hpp',
+    'oatpp/web/FullTest.cpp',
+    'oatpp/web/FullTest.hpp',
+    'oatpp/web/PipelineAsyncTest.cpp',
+    'oatpp/web/PipelineAsyncTest.hpp',
+    'oatpp/web/PipelineTest.cpp',
+    'oatpp/web/PipelineTest.hpp',
+    'oatpp/web/app/BasicAuthorizationController.hpp',
+    'oatpp/web/app/BearerAuthorizationController.hpp',
+    'oatpp/web/app/Client.hpp',
+    'oatpp/web/app/Controller.hpp',
+    'oatpp/web/app/ControllerAsync.hpp',
+    'oatpp/web/app/ControllerWithInterceptors.hpp',
+    'oatpp/web/app/ControllerWithInterceptorsAsync.hpp',
+    'oatpp/web/app/DTOs.hpp',
+    'oatpp/web/mime/multipart/StatefulParserTest.cpp',
+    'oatpp/web/mime/multipart/StatefulParserTest.hpp',
+    'oatpp/web/protocol/http/encoding/ChunkedTest.cpp',
+    'oatpp/web/protocol/http/encoding/ChunkedTest.hpp',
+    'oatpp/web/server/HttpRouterTest.cpp',
+    'oatpp/web/server/HttpRouterTest.hpp',
+    'oatpp/web/server/ServerStopTest.cpp',
+    'oatpp/web/server/ServerStopTest.hpp',
+    'oatpp/web/server/api/ApiControllerTest.cpp',
+    'oatpp/web/server/api/ApiControllerTest.hpp',
+    'oatpp/web/server/handler/AuthorizationHandlerTest.cpp',
+    'oatpp/web/server/handler/AuthorizationHandlerTest.hpp',
+)
+
+
+all_tests = executable(
+    'oatpp_all_tests',
+    test_src_files,
+    cpp_args: flags,
+    include_directories: include_directories('.'),
```

Comment:
You never need to do this. The current source and build directories are put in the search path by default.